index.html

```diff
@@ -0,0 +1,202 @@
+
+                </div>
+            </div>
+            <div class="work-details-row2">
+                <div class="wdr21">
+                    <p>See Project</p>
+                </div>
+                <div class="wdr22">
+                    <h4>Data Dashboard?Healthcare</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+                <div class="wdr23">
+                    <h4>Website Protfolio</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+            </div>
+            <div class="work-details-row3">
+                <div class="wdr31">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+
+                <div class="wdr32">
+                    <h4>Data Dashboard?Healthcare</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+                <div class="wdr33">
+                    <h4>Website Protfolio</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+            </div>
+
+
+        </div>
+
+    </div>
+    <!------------------about me------------------------------------------------>
+    <div class="aboutme-container">
+        <div class="aboutme-row1">
+            <div class="aboutme-left">
+                <h4>About me</h4>
+                <p>Hey there! I can help you build a product, feature or website. Look through some of my work and
+                    experience! If you like
+                    what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <p class="gr">Get My Resume</p>
+            </div>
+            <div class="aboutme-right">
+                <img src="./images/aboutme.png">
```

Comment:
- [ ] As a project requirement you should create the background using CSS tools like `background-image` instead of using `<img>` tag on the HTML file :+1: .